src/sql/optimizer.rs

```diff
@@ -81,8 +79,8 @@ impl DaskSqlOptimizer {
             Arc::new(SimplifyExpressions::new()),
             Arc::new(UnwrapCastInComparison::new()),
             // Arc::new(CommonSubexprEliminate::new()),
-            Arc::new(PushDownProjection::new()),
-            Arc::new(EliminateProjection::new()),
```

Comment:
I think that these two rules were combined into a new `OptimizeProjections` rule, so perhaps we should be using that now?